geoips/xarray_utils/data.py

```diff
@@ -351,46 +363,56 @@ def sector_xarray_spatial(
     for varname in varnames:
         if drop is True:
             if len(sector_inds) == 2:
-                # sector_xarray[varname] = full_xarray[varname].where(xarray_sector_mask, drop=drop)
+                # sector_xarray[varname] =
+                #              full_xarray[varname].where(xarray_sector_mask, drop=drop)
                 sector_xarray[varname] = full_xarray[varname][min_y:max_y, min_x:max_x]
             else:
-                # sector_xarray[varname] = full_xarray[varname].where(xarray_sector_mask, drop=drop)
+                # sector_xarray[varname] =
+                #              full_xarray[varname].where(xarray_sector_mask, drop=drop)
                 sector_xarray[varname] = full_xarray[varname][min_y:max_y]
         else:
             sector_xarray[varname] = full_xarray[varname].where(xarray_sector_mask)
 
-        # To masked array can have a performance hit for dask arrays, do not use them unnecessarily
+        # To masked array can have a performance hit for dask arrays,
+        # do not use them unnecessarily
         # good_speeds = numpy.ma.count(sector_xarray[varname].to_masked_array())
         # if good_speeds > final_good_points:
         #     final_good_points = good_speeds
 
         # if sector_xarray[varname].size < MIN_POINTS or good_speeds < MIN_POINTS:
         #     if verbose:
-        #         LOG.warning('    INSUFFICIENT SPATIAL DATA between %0.2f and %0.2f lon and %0.2f and %0.2f lat, varname %s, %s points',
-        #                     extent_lonlat[0], extent_lonlat[2], extent_lonlat[1], extent_lonlat[3], varname, good_speeds)
+        #         LOG.warning('    INSUFFICIENT SPATIAL DATA between %0.2f and %0.2f lon
+        #                     and %0.2f and %0.2f lat, varname %s, %s points',
+        #                     extent_lonlat[0], extent_lonlat[2], extent_lonlat[1],
+        #                     extent_lonlat[3], varname, good_speeds)
         # else:
         #     if verbose:
-        #         LOG.info('    MATCHED SPATIAL %s points for var %s after location sectoring', good_speeds, varname)
+        #         LOG.info('    MATCHED SPATIAL %s points for var %s after location
+        #                  sectoring', good_speeds, varname)
         #     covg = True
 
     # if covg:
     if sector_xarray["latitude"].size == 0:
         LOG.warning(
-            "    OVERALL INSUFFICIENT SPATIAL DATA between %0.2f and %0.2f lon and %0.2f and %0.2f lat",
+            """    OVERALL INSUFFICIENT SPATIAL DATA between {0} and {1} lon and {2} and
```

Comment:
I don't think we want to do this?  It will actually include all the newlines and white space in the output?

We probably want to break them up onto different lines like
"line1"
"line2"

That should be identical output?

Reply:
Outdated. This is fixed, resolving.